### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by vincentchalamon**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/api-platform/core/pull/1580#issuecomment-355971078
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by soyuka** [comment]
It's only failing on master?

**Comment by vincentchalamon** [comment]
@soyuka Don't think so.
It's related to this: https://github.com/symfony/symfony/commit/0917c4c2ae079ff92c29f5e913b058e96fd5e5b1#diff-9d63a61ac1b3720a090df6b1015822f2

**Comment by soyuka** [comment]
> Maybe should I target to 2.0?

Yes

**Comment by vincentchalamon** [comment]
@soyuka It's rebased on 2.0

**Comment by soyuka** [comment]
still failing with some deprecations :|

**Comment by vincentchalamon** [comment]
@soyuka Those deprecations are out of this PR scope. Was it working earlier? Should I target it on 2.1?

**Comment by Nek-** [comment]
It's failing on master.

File: tests/EventListener/AddFormatListenerTest.php [hunk 1]
```diff
@@ -24,18 +26,18 @@
  */
 class AddFormatListenerTest extends TestCase
 {
-    public function testNoResourceClass()
+    public function testRequestFormatShouldNotBeSet()
     {
-        $request = new Request();
-
+        $requestProphecy = $this->prophesize(Request::class);
```
Comment (dunglas) [inline tests/EventListener/AddFormatListenerTest.php]: Objects (without related interfaces) from external libraries should not be mocked, especially when its a value object like this one.

**Comment by dunglas** [comment]
Thanks @vincentchalamon, fixed in #1634.

**Pull request closed without merging**